content/client-server-api/modules/mentions.md

```diff
@@ -2,26 +2,50 @@
 ### User and room mentions
```

Comment:
I would be curious if this is the right spot to document `m.mentions`?

Reply:
I'd have thought so.